To: vim-dev@vim.org
Subject: Patch 7.3.012
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.012
Problem:    Problems building with MingW.
Solution:   Adjust the MingW makefiles. (Jon)
Files:	    src/Make_ming.mak, src/GvimExt/Make_ming.mak


*** ../vim-7.3.011/src/Make_ming.mak	2010-08-15 21:57:32.000000000 +0200
--- src/Make_ming.mak	2010-09-29 12:32:13.000000000 +0200
***************
*** 303,309 ****
  endif
  endif
  CC := $(CROSS_COMPILE)gcc
! WINDRES := $(CROSS_COMPILE)windres
  
  #>>>>> end of choices
  ###########################################################################
--- 303,309 ----
  endif
  endif
  CC := $(CROSS_COMPILE)gcc
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CC) -E -xc" -
  
  #>>>>> end of choices
  ###########################################################################
*** ../vim-7.3.011/src/GvimExt/Make_ming.mak	2010-08-15 21:57:29.000000000 +0200
--- src/GvimExt/Make_ming.mak	2010-09-29 12:31:07.000000000 +0200
***************
*** 33,39 ****
  endif
  endif
  CXX := $(CROSS_COMPILE)g++
! WINDRES := $(CROSS_COMPILE)windres
  LIBS :=  -luuid
  RES  := gvimext.res
  DEFFILE = gvimext_ming.def
--- 33,39 ----
  endif
  endif
  CXX := $(CROSS_COMPILE)g++
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CXX) -E -xc" -
  LIBS :=  -luuid
  RES  := gvimext.res
  DEFFILE = gvimext_ming.def
*** ../vim-7.3.011/src/version.c	2010-09-21 22:09:28.000000000 +0200
--- src/version.c	2010-09-29 12:36:42.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     12,
  /**/

-- 
If evolution theories are correct, humans will soon grow a third
hand for operating the mouse.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
